### PR TITLE
Add access-your-teaching-qualifications domain

### PR DESCRIPTION
**Pull request by NickGraham101**
## Context

New service access-your-teaching-qualifications requires a DNS zone and FrontDoor profile

## Changes in this PR

- Adds DNS zone access-your-teaching-qualifications.education.gov.uk
- Adds FrontDoor profile
- Replaced script that creates resource group, storage account and key vault with the make recipe now in use on other TRA repos

## Guidance to review

The terraform will be ran locally using `make aytq domains-infra-apply` and `make aytq dev domains-apply`, please check config is correct.

### CODE REVIEW

**Comment by NickGraham101** [comment]
[Infrastructure and domain naming](https://trello.com/c/eT6yaaHD/694-infrastructure-and-domain-naming)

File: Makefile [hunk 1]
```diff
@@ -11,50 +11,72 @@ dev:
...
 	$(eval DOMAINS_ID=getanid)
+	$(eval PRODUCT_NAME="Get an Identity")
 
 aytp:
 	$(eval DOMAINS_ID=aytp)
+	$(eval PRODUCT_NAME="Access Your Teaching Profile")
+
+aytq:
+	$(eval DOMAINS_ID=aytq)
+	$(eval PRODUCT_NAME="Access Your Teaching Certificates")
```
Comment (saliceti) [inline Makefile]: Isn't it "Qualifications"?

File: custom_domains/aytq/terraform.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  features {}
+
+  skip_provider_registration = true
+}
+
+provider "azurerm" {
+  features {}
+  version         = "3.45.0"
+  alias           = "app_subcription"
+  subscription_id = var.app_sub_id
```
Comment (saliceti) [inline custom_domains/aytq/terraform.tf:24]: This may not work in the workflow since the service principal is tied to one subscription

Comment (saliceti) [reply custom_domains/aytq/terraform.tf:24]: As discussed IRL, this is deployed manually, not via pipeline

**Comment by nickdevdfe** [comment]
All AYTQ environments successfully deployed. This PR is now ready to be reviewed and merged.

File: Makefile [hunk 3]
```diff
@@ -6,75 +6,100 @@ help: ## Show this help
 	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: dev
-dev:
+dev: ## set the dev enironment variables
```
Comment (saliceti) [inline Makefile]: Typo: enironment

**Review by saliceti — APPROVED** [review]

**Review by saliceti — APPROVED** [review]

**Pull request merged**